core/src/main/java/org/testcontainers/containers/image/ImageData.java

```diff
@@ -0,0 +1,23 @@
+package org.testcontainers.containers.image;
+
+import java.time.Instant;
+import java.util.List;
+
+public interface ImageData {
+
+    Instant getCreated();
+
+    String getId();
+
+    String getParentId();
+
+    List<String> getRepoTags();
+
+    List<String> getRepoDigests();
+
+    Long getSize();
```

Comment:
Is it nullable? If not, I would suggest `long` instead of `Long`. If yes, `OptionalLong`.

Reply:
Yes, it is nullable. Changed to OptionalLong.

Reply:
CI still fails unfortunately :(